command-line/dot-profile.md

```diff
@@ -12,7 +12,10 @@ Here are links to lessons that should be completed before this lesson:
 
 ### Motivation
 
-Learn about how to customize a computer's environment 
+Learn about how to customize a computer's environment .
```

Comment:
Remove extra space before period.

Reply:
This syntax doesn’t fit the section. Maybe:

With knowledge about dot profile, you can customize your computer’s environment.